qiskit/schemas/qobj_schema.json

```diff
@@ -847,6 +847,41 @@
             },
             "type": "array"
         },
+        "calibration_entry": {
```

Comment:
The `"version"` field at the top of this file has to be bumped as well. Right?

Reply:
thank you!